src/components/MessageCard/MessageCard.style.ts

```diff
@@ -1,7 +1,7 @@
 import { Box } from '@mui/material';
 import { alpha, darken, lighten, styled } from '@mui/material/styles';
 
-const MessageCard = styled(Box)(({ theme }) => ({
+const MessageCard = styled(Box)(({ theme, style }) => ({
```

Comment:
Please don't add additional `style` props, MUI provides us with the `sx` property if we would like to override styles.